### DISCUSSION TITLE
Best Practices For CTRL+R or F5

### DISCUSSION THREAD

**Discussion by tbreuss** [unpoly/unpoly#441] [Q&A]
We are evaluating unpoly for a next bigger project and stumbled across the following behavior.

When we are on a subpage (or layer) and the browser is reloaded via CTRL+R or F5 the context is lost and only the subpage (or layer) embedded into the layout is displayed. This seems to be the normal behavior and therefore is also seen in the official unpoly demo.

Nevertheless, I want to ask the question, whether there is some kind of work-around or best practice for this behavior.

**Comment by triskweline** [comment]
Unpoly encourages every server response to send a full page. Hence content displayed in overlays are also expected to work when shown as a full page. [Subinteractions](https://unpoly.com/subinteractions) allows pages to work in both modes, without additional code.

As an alternative you can disable history for modal overlays:

```js
up.layer.config.modal.history = false
```

**Reply by tbreuss** [reply]
Thanks for the answer, although my question was quite unclear. Sorry for that.

I am working now with Unpoly on an admin interface, the experience so far ist just great. 

Thank you very much for your work and this library!
